WPMedia/fusion-news-theme-blocks
# TMEDIA-250 - Card List Block Story

## Description
Add stories for Card List Block
* Refactored to use functional components to make it easier to mock within storybook

## Jira Ticket
- [TMEDIA-250](https://arcpublishing.atlassian.net/browse/TMEDIA-250)

## Test Steps

1. Checkout this branch `git checkout TMEDIA-250-card-list-story`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/card-list-block`
3. Verify Card List block works as before - no changes in functionality
 
## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
